lib/jsonapi/utils/support/pagination.rb

```diff
@@ -139,6 +139,16 @@ def count_records(records, options)
             records.except(:group, :order).count("DISTINCT #{records.table.name}.id")
           end
         end
+
+        def count_pages(records, options)
```

Comment:
Here it actually won't need the `options` argument, so it can be removed. Also, since the first argument should be changed from an array to an integer (the count itself) its name might get a bit clearer if changed to `record_count`.